load-test/README.md

```diff
@@ -0,0 +1,44 @@
+# Load Tests
+
+This package contains code for load testing specific endpoints of ZITADEL using [k6](https://k6.io).
+
+## Prerequisite
+
+* [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+* [k6](https://k6.io/docs/get-started/installation/)
+* [go](https://go.dev/doc/install)
+* running ZITADEL
+
+## Structure
+
+The use cases under tests are defined in `src/use_cases`. The implementation of ZITADEL resources and calls are located under `src`.
+
+## Execution
+
+### Env vars
+
+`VUS`: Amount of parallel processes execute the test (default is 20)
+`DURATION`: Defines how long the tests are executed (default is `200s`)
+`ZITADEL_HOST`: URL of ZITADEL (default is `http://localhost:8080`)
+
+To setup the tests we use the credentials of console and log in using an admin. The user must be able to create organizations and all resources inside organizations.
+
+`ADMIN_LOGIN_NAME`: `zitadel-admin@zitadel.localhost`
+`ADMIN_PASSWORD`: `Password1!`
+
+### Test
+
+* `make human_password_login`  
+  setup: creates human users  
+  test: uses the previously created humans to sign in using the login ui
```

Comment:
I created a new IAM user in the console, but didn't login with it before I ran the loadtest. I guess we have to document that the user must have passed first login, so that MFA and password prompts won't show anymore:

```
ERRO[0003] Uncaught (in promise) GoError: wrong password callback: http://localhost:9000/ui/login/mfa/prompt
        at go.k6.io/k6/js/modules/k6.(*K6).Fail-fm (native)
        at passwordCallback (webpack://typescript/./src/login_ui.ts:73:97(22))
        at go.k6.io/k6/js/modules/k6.(*K6).Check-fm (native)
        at enterPassword (webpack://typescript/./src/login_ui.ts:71:8(44))
        at loginByUsernamePassword (webpack://typescript/./src/login_ui.ts:19:41(24))
        at _callee$ (webpack://typescript/./src/use_cases/human_password_login.ts:9:41(45))
        at call (native)
        at tryCatch (webpack://typescript/./src/use_cases/human_password_login.ts:2:0(9))
        at webpack://typescript/./src/use_cases/human_password_login.ts:2:0(103)
        at webpack://typescript/./src/use_cases/human_password_login.ts:2:0(5)
        at asyncGeneratorStep (webpack://typescript/./src/use_cases/human_password_login.ts:2:0(6))
        at _next (webpack://typescript/./src/use_cases/human_password_login.ts:2:0(10))
        at webpack://typescript/./src/use_cases/human_password_login.ts:2:0(14)
        at webpack://typescript/./src/use_cases/human_password_login.ts:2:0(9)
        at apply (native)
        at _setup (webpack://typescript/./src/use_cases/human_password_login.ts:24:1(17))
        at apply (native)
        at setup (webpack://typescript/./src/use_cases/human_password_login.ts:8:27(7)) 
make: *** [Makefile:9: human_password_login] Error 255
```